pages/index.tsx

```diff
@@ -1,9 +1,34 @@
 import type { NextPage } from "next";
 import { ThemeProvider } from "@mui/material";
 import { theme } from "../styles/theme";
+import { Divider } from "@mui/material";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
 
 const Home: NextPage = () => {
-  return <ThemeProvider theme={theme}></ThemeProvider>;
+  return (
+    <ThemeProvider theme={theme}>
+      <div className="tool-box">
+        <h3>Tools</h3>
+        <Divider light />
+        <Typography variant="body1" component="p">
+          🛠 Internal tools + useful things for core members like…
+        </Typography>
+        <Typography variant="body1" component="ul">
+          <li>
+            <Link
+              href="https://tools.hackbeanpot.com/sigmaker/"
+              target="_blank"
```

Comment:
Let's remove this since we're opening a link to an internal page and don't need to open a new tab!